### DISCUSSION TITLE
"banger" -- correct values for trigger bangs across acts?

### DISCUSSION THREAD

**Discussion by jpsteccato** [ppooll-dev/ppooll#130] [General]
hi @klausfilip !   

I'm working on some acts that are meant to connect to banger & control@ to "trigger" different events across ppooll, but i'm having trouble figuring out what the actual values should be to consistently receive/send bangs from acts to other acts with "trigger" (T) buttons.

is there a certain way these bangs should be sent across acts in ppooll?  

here's an example screenshot: when a bang shows in "control@", it seems to send a -1 on every bang, but the "bang!" control mode doesn't seem to do anything.  i decided to use valON like in the screenshot, and this seems to do what I want

<img width="723" alt="image" src="https://github.com/ppooll-dev/ppooll/assets/37079335/4fc15c9b-683d-4082-9001-e074ea819e35">

**Comment by klausfilip** [comment]
well banger. 
it sends -1 values to control@ because buttons in button-mode can be triggered like this.
(just found out the text-button behaviour changed, so that now any value will trigger them)
so you also might leave the mode at its default scale, for a button.
if you want to send a value to another parameter valON seems a good choice.

anyway, the bang! mode in control@ seems to be completely useless.
